clever deploy exits with "couldn't find a domain name" with an application without domains

Running `clever deploy --follow` might exit with code 1 when an application has no domain names.

```
2025-11-18T17:42:10.828Z: Successfully deployed in 0 minutes and 41 seconds
[ERROR] Couldn't find a domain name
```

Not having domain names shouldn't quit the deploy command.

--- a/src/models/log-v4.js
+++ b/src/models/log-v4.js
@@ -134,11 +134,17 @@ export async function watchDeploymentAndDisplayLogs(options) {
   }
 
   if (deploymentEnded.state === 'OK') {
-    const favouriteDomain = await getBest(appId, ownerId);
     Logger.println('');
-    Logger.println(
-      `${styleText(['bold', 'green'], '✓ Access your application:')} ${styleText(['underline', 'bold'], `https://${favouriteDomain.fqdn}`)}`,
-    );
+
+    // There can be applications without any domain, so we don't fail if we can't get one
+    const favouriteDomain = await getBest(appId, ownerId).catch(() => null);
+
+    if (favouriteDomain) {
+      Logger.println(
+        `${styleText(['bold', 'green'], '✓ Access your application:')} ${styleText(['underline', 'bold'], `https://${favouriteDomain.fqdn}`)}`,
+      );
+    }
+
     Logger.println(
       `${styleText(['bold', 'blue'], '→ Manage your application:')} ${styleText(['underline', 'bold'], `${conf.GOTO_URL}/${appId}`)}`,
     );

